### PR TITLE
NXP-26890: make the selectivity setter helper to clear value in single mode

### PR DESCRIPTION
**Pull request by FabioFSantos**

### CODE REVIEW

**Comment by FabioFSantos** [comment]
https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/357/

File: ftest/web-ui/webdriver/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "test": "nuxeo-web-ui-ftest --report --screenshots --headless",
     "test:dev": "nuxeo-web-ui-ftest --nuxeoUrl=http://localhost:8080/nuxeo --url=http://localhost:5000/",
-    "test:watch": "nuxeo-web-ui-ftest --nuxeoUrl=http://localhost:8080/nuxeo  --url=http://localhost:5000/ --tags=@watch --debug"
+    "test:watch": "nuxeo-web-ui-ftest --nuxeoUrl=http://localhost:8080/nuxeo --tags=@watch --debug"
```
Comment (nmpcunha) [inline ftest/web-ui/webdriver/package.json]: Despite the comments done on https://github.com/nuxeo/nuxeo-web-ui/pull/806, we should not change this parameter

Comment (FabioFSantos) [reply ftest/web-ui/webdriver/package.json]: wasn't supposed to be erased, it's back to normal 👍

**Comment by FabioFSantos** [comment]
https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/362/

File: ftest/web-ui/webdriver/test/features/metadata.feature [hunk 2]
```diff
@@ -49,11 +49,12 @@ Feature: Edit metadata
       | nature       | Internship report                       |
       | subjects     | Science/Medicine,Daily life/Video games |
     Then I can edit the following properties in the File metadata:
-      | name         | value |
-      | nature       |       |
-      | subjects     |       |
+      | name         | value    |
+      | nature       |          |
+      | subjects     |          |
+      | coverage     | Portugal |
```
Comment (nmpcunha) [inline ftest/web-ui/webdriver/test/features/metadata.feature]: Should this line be removed? We are not using `coverage` anywhere else.

**Review by nmpcunha — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**